saylua/modules/pets/views.py

```diff
@@ -1,18 +1,83 @@
-from flask import render_template, redirect
+from flask import render_template, g, redirect, request, flash
+from saylua import db
+from .models.db import Pet
 
 
 def pet_profile(name):
```

Comment:
There is way too much redundancy here, if you're performing the same action 3+ times why not just construct an output string and add / commit / flash all at once at the end?

I would like to see this at least partially simplified for somebody's sake further down the road